### PR TITLE
new shit tweak (like clockwork)

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
after being merged for awhile, the two new virus symptoms need a few small tweaks, so here's a half-hour pr with them
-Biometallic replication is now a bit better stat wise, and will buff pre-existing robotic limbs as well. This is so IPCS can get a decent trifecta
-exolocomotive xenomitosis now turns more shit pink- zombies and macrophages are both themed with the virus
-TRD now heals mechanical bodyparts as intended

## Why It's Good For The Game
lol tubby custard

## Changelog
:cl:
tweak: biometallic replication has slightly better stats, buffs preexisting robot limbs
fix: TRD now works on mechanical limbs as intended
fix: fixed exolocomotive xenomitosis capitalization and healing threshold
tweak: exolocomotive xenomitosis now works on monky
imageadd: pink zombies and macrophages, so exolocomotive xenomitosis can be more aesthetic
/:cl:

### CODE REVIEW

**Comment by LordSaladin** [comment]
yes.

**Review by PowerfulBacon — DISMISSED** [review]
pink ian when?

File: code/datums/diseases/advance/symptoms/heal.dm [hunk 1]
```diff
@@ -382,7 +382,7 @@ obj/effect/sweatsplash/proc/splash()
 	switch(A.stage)
 		if(4, 5)
 			if(burnheal)
-				M.adjustFireLoss(-1 * power) //we don't care about bodypart type here, because this is rolling back time or some shit
+				M.heal_overall_damage(0, 1) //no required_status checks here, this does all bodyparts equally
```
Comment (PowerfulBacon) [inline code/datums/diseases/advance/symptoms/heal.dm:385]: Would be nicer / more obvious what this was doing exactly with
M.heal_overall_damage(burn=1) //no required_status checks here, this does all bodyparts equally

Comment (zeskorion) [reply code/datums/diseases/advance/symptoms/heal.dm:385]: adjustfireloss automatically applies a required_status of BODYPART_ORGANIC to bodyparts. heal overall damage doesnt, so it heals robotic limbs

**Comment by zeskorion** [comment]
one sec, need to make another commit here

**Comment by zeskorion** [comment]
done

also,
> pink ian when?

I could actually make this virus add ian to viable mobtypes, infect him, and turn him into a pink corgi.  i have no idea if itd work or not, but most virus shit only uses isliving()

**Review by PowerfulBacon — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]
pink zombies based

**Comment by PowerfulBacon** [comment]
Enjoy the double approval

**Pull request merged**